stages/org.osbuild.ostree.deploy-verity

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+"""Enable fs-verity on an ostree repo/deployment
+
+The way osbuild works the ostree deployment is built in a chroot and
+stored as a regular directory of files before finally being copied to
+the physical filesystem. This means ostree fs-verity support doesn't
+work, as the fs-verity setting of files is not copied.
```

Comment:
Wait is this because osbuild doesn't use `ostree pull-local` but instead just copies things via e.g. `tar` or something?

I think that's what we want to fix, so that ostree is in charge of writing to the disk, not osbuild.

Reply:
As I said above, that just doesn't work with the way osbuild is architected.